x-pack/docs/en/rest-api/security/create-role-mappings.asciidoc

```diff
@@ -37,6 +37,31 @@ roles API>> or <<roles-management-file,roles files>>.
 
 For more information, see <<mapping-roles>>.
 
+[[security-api-put-role-mapping-desc-templates]]
```

Comment:
It's a bad practice to link to the auto-generated anchors, so it's unfortunate that one in the Kibana docs snuck through.  I think by explicitly defining that anchor, however, we should avoid breaking the link:
```suggestion
[[_role_templates]]
```

Reply:
Thanks Lisa. This is a clever way to solve the issue.